shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -74,6 +81,8 @@ typedef struct {
   // Array of Dart entrypoint arguments. This is deep copied during the call
   // to FlutterDesktopRunEngine.
   const char** dart_entrypoint_argv;
+  // The renderer type of the engine.
+  FlutterDesktopEngineRendererType renderer_type;
```

Comment:
Please consider moving this property to `FlutterDesktopView(Window)Properties` since it's not relevant for headless engine. You will need to refactor the code a little bit, for example,
1. Move the renderer initialization code from `FlutterTizenEngine`'s constructor to a separate method and call it from `FlutterTizenView`'s `SetEngine` or `CreateRenderSurface`, or
2. Let `FlutterTizenView` own the renderer instance.

Note: `event_loop_` and `render_loop_` can be initialized during `RunEngine`.

Does this make sense? Do you have any thought? @bbrto21

Reply:
How about adding a `None` type in Type for headless?

I can move the renderer to FlutterTizenView as you said.
However, when FlutterEngine used in `flutter-tizen`, I think that if we create a renderer in the engine, it can be helpful for improvement of the launch time(?).

Reply:
You guys are right. This way is a problem because the render type must be set when creating a FlutterEngine for ElmFlutterView.
I will move this to the view. I think there will be more code changes...

Reply:
Is the `renderer_type` property only applicable to `FlutterDesktopWindowProperties`? How about `FlutterDesktopViewProperties`?

Regarding the launch time optimization, maybe we could remove `RunEngine` from the `FlutterDesktopViewCreate` family and provide another API that starts the engine associated with a view.

Reply:
I added renderer_type only to FlutterDesktopWindowProperties. I didn't add it to FlutterDesktopViewProperties because for now only EvasGL is used. We can add them later if needed.